src/app/ui/TaskBoard.tsx

```diff
@@ -39,7 +40,7 @@ export const TaskBoard = ({
     store.dispatch(
       updateWorkflowStateIdByTaskId({ taskId: payload.taskId, targetWorkflowStateId: payload.targetWorkflowStateId }),
     )
-    updateWorkflowStateIdOfTask(payload.taskId, payload.targetWorkflowStateId)
+    updateTask(payload.taskId, { workflowStateId: payload.targetWorkflowStateId })
   }, [])
```

Comment:
Please fix lint warning regarding useEffect dependency

Reply:
Added in bc28efa0b4fcd297a63bb056c9756da3bf55b9c7 commit